src/winzone/WindowsTimeZoneIDs.jl

```diff
@@ -46,8 +49,15 @@ function build(xml_file::AbstractString=WINDOWS_XML_FILE; force::Bool=false)
         if isfile(fallback_xml_file) && !force
             cp(fallback_xml_file, xml_file)
         else
-            @info "Downloading latest Windows to POSIX timezone ID XML"
-            download(WINDOWS_ZONE_URL, xml_file)
+            if VERSION >= v"1.4"
```

Comment:
Causing CI tests to fail on earlier versions of Julia
```suggestion
            @static if VERSION >= v"1.4"
```